.vscode/settings.json

```diff
@@ -14,8 +14,8 @@
   "files.trimFinalNewlines": true,
   "files.trimTrailingWhitespace": true,
   "editor.rulers": [
-    80,
-    90
+    90,
+    100
```

Comment:
Not super important, but why not use 80,100? I always forget if we have an agreed upon and written down style guide for Bittide specifically, but the Clash style guide also says to keep below 80 and hard below 90

Reply:
Yeah that's good too!